blog/content/post/2020-04-30-ipxe-https.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Enabling HTTPS support in iPXE
```

Comment:
@miczyg1 adding `User friendly tutorial for enabling HTTPS support in iPXE` to the headline boosts the https://www.aminstitute.com/process/headline.cgi rating from 0% to 44.44%, maybe we can increase it even more?

Reply:
Applied your suggestion